src/modules/mount/mount.conf

```diff
@@ -19,6 +19,10 @@
 # The device is not mounted if the mountPoint is unset or if the fs is
 # set to unformatted.
 #
+---
+# Btrfs subvolumes to create if root filesystem is on btrfs volume.
+# If mountpoint is mounted already to another partition, it is ignored.
+# Separate subvolume for swapfile is handled separately and automatically.
```

Comment:
This comment should probably be with the configuration key it describes, not here.

Reply:
Done